Fhi.Smittestopp.Verification.Tests/Msis/MsisClientTests.cs

```diff
@@ -0,0 +1,101 @@
+                .SetupRequest(HttpMethod.Get, "/v1/Msis/erMsisOnline", new HttpResponseMessage
+                {
+                    StatusCode = HttpStatusCode.OK,
+                    Content = new StringContent(responseBody, Encoding.UTF8, "application/json"),
+                });
+            var httpClient = new HttpClient(handlerMock.Object)
+            {
+                BaseAddress = new Uri("http://test.com/v1/Msis/"),
+            };
+
+            var target = new MsisClient(httpClient);
+
+            // ACT
+            var result = await target.GetMsisOnlineStatus();
+
+            // ASSERT
+            result.Should().Be(expectedResult);
+            handlerMock.Verify();
+        }
+
+        [Test]
+        public async Task GetCovid19Status_GivenPositiveResponse_ReturnsPositiveResult()
+        {
+            // ARRANGE
+            var handlerMock = new Mock<HttpMessageHandler>()
+                .SetupRequest(HttpMethod.Get, "/v1/Msis/covid19status?ident=01019098765", new HttpResponseMessage
+                {
+                    StatusCode = HttpStatusCode.OK,
+                    Content = new StringContent(@"{
+                        ""harPositivCovid19Prove"": true,
+                        ""provedato"": ""2020-11-19T17:08:59.902Z""
+                    }", Encoding.UTF8, "application/json"),
+                });
+            var httpClient = new HttpClient(handlerMock.Object)
+            {
+                BaseAddress = new Uri("http://test.com/v1/Msis/"),
+            };
+
+            var target = new MsisClient(httpClient);
+
+            // ACT
+            var result = await target.GetCovid19Status("01019098765");
+
+            // ASSERT
+            result.HarPositivCovid19Prove.Should().BeTrue();
+            result.Provedato.Should().BeCloseTo(DateTime.Parse("2020-11-19T17:08:59.902Z").ToUniversalTime());
+            handlerMock.Verify();
+        }
+
+        [Test]
+        public async Task GetCovid19Status_GivenNonPositiveResponse_ReturnsNonPositiveResult()
+        {
+            // ARRANGE
+            var handlerMock = new Mock<HttpMessageHandler>()
+                .SetupRequest(HttpMethod.Get, "/v1/Msis/covid19status?ident=01019098765", new HttpResponseMessage
+                {
+                    StatusCode = HttpStatusCode.OK,
+                    Content = new StringContent(@"{
+                        ""harPositivCovid19Prove"": false,
+                        ""provedato"": null
+                    }", Encoding.UTF8, "application/json"),
+                });
+            var httpClient = new HttpClient(handlerMock.Object)
+            {
+                BaseAddress = new Uri("http://test.com/v1/Msis/"),
+            };
+
+            var target = new MsisClient(httpClient);
+
+            // ACT
+            var result = await target.GetCovid19Status("01019098765");
+
+            // ASSERT
+            result.HarPositivCovid19Prove.Should().BeFalse();
```

Comment:
Could we use an Assert.Multiple lambda expression here, to do the tests in parallell?